app/javascript/components/StyleFoundation/Box.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+import * as R from 'ramda'
+
+const toPixelIfDefined = (size) => !R.isNil(size) && `${size}px`
```

Comment:
it might be more flexible to let the caller decide on the unit and default to `px` when not specified.